setup.py

```diff
@@ -21,24 +21,24 @@
                       'ipympl>=0.7.0,<0.8.0',
                       'jupyter>=1.0.0,<2',
                       'jupyter_contrib_nbextensions>=0.5.1,<1',
-                      'jupyterlab>=3.1.5,<4',
+                      'jupyterlab>=3.4.3,<4',
                       'matplotlib>=2.2.2,<3',
-                      'numpy>=1.16.3,<2',
+                      'numpy>=1.21.6,<2',
                       'palettable>=3.3.0,<4',
-                      'pandas>=0.23.3,<1',
-                      'requests>=2.25.1,<3',
+                      'pandas>=1.3.5,<2',
+                      'requests>=2.27.1,<3',
                       'scikit-image>=0.14.3,<=0.16.2',
-                      'scikit-learn>=0.19.1,<1',
-                      'scipy>=1.1.0,<2',
-                      'seaborn>=0.10.1,<1',
+                      'scikit-learn>=0.24.2,<1',
+                      'scipy>=1.7.3,<2',
+                      'seaborn>=0.11.2,<1',
                       'spatial-lda>=0.1.3,<1',
-                      'statsmodels>=0.11.1,<1',
-                      'umap-learn>=0.4.6,<1',
-                      'xarray>=0.12.3,<1',
-                      'tqdm>=4.54.1,<5'],
-    dependency_links=[
-        'git+https://github.com/calico/spatial_lda.git@primary',
-    ],
+                      'statsmodels>=0.13.2,<1',
+                      'umap-learn>=0.5.3,<1',
+                      'xarray>=0.17.0,<1',
+                      'tqdm>=4.64.0,<5'],
+    # dependency_links=[
+    #     'git+https://github.com/calico/spatial_lda.git@primary',
```

Comment:
If it's up to date, we can remove this (I think?). If not we'll keep everything linked to the github

Reply:
Yes, this indeed can be removed.